go.mod

```diff
@@ -82,6 +82,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0 // indirect
 	github.com/opencontainers/image-spec v1.1.0-rc2 // indirect
 	github.com/opencontainers/runc v1.1.4 // indirect
+	github.com/package-url/packageurl-go v0.1.0 // indirect
```

Comment:
When I run `go mod tidy` this line disappears, along with the equivalent in `go.sum`.

This makes senes to me, as the use of the `packageurl-go` library is in the `dependency` package which has its own go.mod file - we haven't added this package to the buildpack itself.

Could you validate this by running `go mod tidy` on your local copy and seeing if you get the same result?

Reply:
I get the same result. I'll resolve that in a subsequent commit.